models/classes/LtiProvider/LtiProvider.php

```diff
@@ -110,6 +97,8 @@ public function getLtiVersion(): string
 
     public function getToolClientId(): string
     {
+        //My instance to properly work has to have this equal to any lti providers
```

Comment:
I think this comment is unnecessary. Can you remove, please? The client_id should work. If it is not, something is wrong